### PR TITLE
Bring in the new SPO category from revamp branch

### PR DESCRIPTION
**Pull request by katomm**
This is a draft to bring in the new SPO category from revamped branch 
(this replaces #1061)

### CODE REVIEW

**Review by rphair — APPROVED** [review]
This looks fine, and long awaited 🥳 ... but currently can't test & review in local build because for some reason `yarn build` won't fetch the CIP matter after finally getting broadband: maybe an IX level problem with my new ISP that could take a while to solve, but if I can fix it before this is merged I'll re-check in a local build. 🤓

**Comment by katomm** [comment]
Thanks @rphair. Build log also available here: https://github.com/cardano-foundation/developer-portal/actions/runs/4978308037/jobs/8908435772 🤓

**Review by rdlrt — APPROVED** [review]

**Review by gufmar — APPROVED** [review]
LGTM (may clean some Ticker traces. We should not open precedence cases)

File: static/img/stake-pool-guide/snsky_dashboard.jpg
Comment (katomm) [reply static/img/stake-pool-guide/snsky_dashboard.jpg:1]: Good catch

**Comment by rphair** [comment]
yes @katomm the build logs are great, but whenever possible I try to look at the locally built web site when 57 files are changed & the whole outline structure is rearranged 🤩

Note my local build still can't fetch the CIP content, upon which the rest of the local build depends.  My geolocation or IP range is apparently being blocked by the CDN which is serving that content.  Can someone familiar with the build scripts give me an idea how to pursue this?  cc @oversize @Mercurial @rdlrt @os11k

**Comment by rphair** [comment]
Just in case anyone has seen this before: my ISP is blocking raw.githubusercontent.com ... or maybe GitHub is blocking my ISP's addresses for that subdomain.  It affects other GitHub procedures besides the CIP content download (e.g. any time you hit the raw button & a lot of linked content for the dev portal & CIP repo).

This is my first really active year on GitHub & maybe someone with longer experience has seen this before & might know who at GitHub might help follow up about it. 😕🙏

**Comment by rdlrt** [comment]
Well might be something we could discuss elsewhere - but since you mention subdomain - if the issue is with DNS CNAME resolution, I'd suggest try using alternate DNS (cloudflare/google/opendns being most popular ones) or a proxy (or simpler shell scripts to update hosts file mapping with nslookup or IPs fetched from elsewhere)

**Comment by os11k** [comment]
DNS-over-TLS + cloudlfare is my preferred setup.

**Pull request merged**
